spacy/pipeline/coref.py

```diff
@@ -511,15 +509,20 @@ def update(
         set_dropout_rate(self.model, drop)
 
         total_loss = 0
-
-        for eg in examples:
-            preds, backprop = self.model.begin_update([eg.predicted])
-            score_matrix, mention_idx = preds
-
-            loss, d_scores = self.get_loss([eg], score_matrix, mention_idx)
+        docs = [eg.predicted for eg in examples]
+        for doc, eg in zip(docs, examples):
```

Comment:
Minor point but can you do something like this instead:

```
for eg in examples:
    doc = eg.predicted
```

Reply:
Ah, I realize this is the same thing I'm doing in the EntityLinker :sweat_smile: It's fine as is, might be cleaner with the above pattern.

Reply:
I did re-write it though :D